dali/operators/image/peek_shape/peek_image_shape.h

```diff
@@ -62,10 +64,9 @@ class PeekImageShape : public Operator<CPUBackend> {
   }
 
   template <typename type>
-  void WriteShape(Tensor<CPUBackend> &out, const TensorShape<3> &shape) {
-    type *data = out.mutable_data<type>();
+  void WriteShape(TensorView<StorageCPU, type, 1> out, const TensorShape<3> &shape) {
```

Comment:
Not related to this PR, but it could be a template against `ndim` and use a run-time number of dimensions in the loop. This would help should we ever consider extending the functionality to volumes.